modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+OpenShift API for Data Protection (OADP) is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate data from one {product-title} cluster to another or to migrate data from a VMware cluster to an {product-title} cluster.
+
+You can use OADP to back application data from one cluster and restore it on another. However, doing so is both more complicated than using {mtc-short} or using OADP to back up and restore to the same {product-name} cluster.
+
+== Considerations in using OADP to back up application data from one cluster and restore it to another
+
+Before you begin planning to use OADP to back up application data from one cluster and restore it to another cluster, consider the following limitations:
+
+* You must exclude Operators from backup and restore operations because OADP is not designed to either back up or restore Operators.
+* Velero, which OADP is built upon, does not natively support the migration of persistent volumes snapshots across cloud providers. To migrate volume data between cloud platforms, you must enable Velero's File System Backup, which backs up volume contents at the filesystem level.
```

Comment:
TODO: add/link to how section

Reply:
@kaovilai @weshayutin I need advice on what to include from https://velero.io/docs/main/file-system-backup/. Wes has said in general that we need to modify Velero instructions so please advise what to include and exclude and any modifications needed.

Reply:
@kaovilai re: "I think...out of scope" -- wasn't sure if you wanted that comment incorporated into the text here.

Reply:
Not into the texts just saying what's in this PR (updating the word restic to file system backup) may not fit the ask from Jira. Perhaps there's another Jira capturing this task that can be additionally linked to the PR.